dotcom-rendering/cdk/lib/dotcom-rendering.ts

```diff
@@ -146,6 +147,13 @@ export class DotcomRendering extends GuStack {
 		new CfnOutput(this, 'LoadBalancerUrl', {
 			value: loadBalancer.loadBalancerDnsName,
 		});
+
+		new StringParameter(this, 'ec2RoleArn', {
+			// Annoyingly this doesn't follow the same pattern as the other SSM parameters
+			parameterName: `/${stack}/${stage}/${app}.loadBalancerDnsName`,
+			stringValue: loadBalancer.loadBalancerDnsName,
+		});
```

Comment:
This autogenerates the SSM parameter that we can use [here](https://github.com/guardian/frontend/pull/26556/files#diff-e4a4c985ae2e37d93fe50a7627d4a1d0912773c3b1844b8b2bb8cbd1cf78da16R597).